### PR TITLE
Update SonarQube Docs

### PR DESCRIPTION
**Pull request by Felixnoo**
Signed-off-by: Felixnoo <felixliu@yunify.com>

### CODE REVIEW

**Comment by Felixnoo** [comment]
/assign @JohnNiang 
Please take a look at the updates I made. Thanks.

File: content/zh/docs/devops-user-guide/how-to-integrate/sonarqube.md [hunk 1]
```diff
@@ -83,9 +83,8 @@ weight: 11310
    s2ioperator-0                              1/1     Running   1          84m
    sonarqube-postgresql-0                     1/1     Running   0          5m31s
    sonarqube-sonarqube-bb595d88b-97594        1/1     Running   2          5m31s
-   uc-jenkins-update-center-8c898f44f-m8dz2   1/1     Running   0          85m
    ```
-
+
```
Comment (JohnNiang) [inline content/zh/docs/devops-user-guide/how-to-integrate/sonarqube.md:87]: please remove useless three spaces.

File: content/en/docs/devops-user-guide/how-to-integrate/sonarqube.md [hunk 2]
```diff
@@ -83,9 +83,8 @@ To integrate SonarQube into your pipeline, you must install SonarQube Server fir
    s2ioperator-0                              1/1     Running   1          84m
    sonarqube-postgresql-0                     1/1     Running   0          5m31s
    sonarqube-sonarqube-bb595d88b-97594        1/1     Running   2          5m31s
-   uc-jenkins-update-center-8c898f44f-m8dz2   1/1     Running   0          85m
    ```
-
+
```
Comment (JohnNiang) [inline content/en/docs/devops-user-guide/how-to-integrate/sonarqube.md:87]: please remove useless three spaces.

**Review by JohnNiang — CHANGES_REQUESTED** [review]

File: content/zh/docs/devops-user-guide/how-to-integrate/sonarqube.md (hunk 1 above)
Comment (Felixnoo) [reply content/zh/docs/devops-user-guide/how-to-integrate/sonarqube.md]: Removed.

File: content/en/docs/devops-user-guide/how-to-integrate/sonarqube.md (hunk 2 above)
Comment (Felixnoo) [reply content/en/docs/devops-user-guide/how-to-integrate/sonarqube.md]: Removed.

**Comment by Felixnoo** [comment]
/cc @kubesphere/sig-devops 
FYI.

**Pull request merged**